src/components/auth/Login.jsx

```diff
@@ -32,7 +84,7 @@ export const Login = () => {
    return (
       <form onSubmit={handleSubmit(onSubmit)}>
          <div className="flex flex-col my-1">
-            {/* email */}
+            {alertMessage.message && <Alert alertMessage={alertMessage} />}
```

Comment:
avoid using && for returning JSX, it can lead to undesired behaviors, instead use the ternary operator

i.e.
condition ? Component : null